travis/install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+source "${0%/*}/setup.sh"
+
+time CMAKE_ARGS='-DUSE_ATEN=ON -DUSE_OPENMP=ON' "$top_dir/install-develop.sh"
```

Comment:
onnx-coreml's caffe2 use case doesn't need ATEN (and OPENMP).

Reply:
Fixed